tutorials/3d/standard_material_3d.rst

```diff
@@ -414,6 +419,43 @@ and wider compatibility.
   popular engines) can be found
   `here <http://wiki.polycount.com/wiki/Normal_Map_Technical_Details>`__.
 
+Bent normal map
+---------------
+
+A bent normal map describes the average direction of ambient lighting. Unlike a
+regular normal map, this is used to improve how a material reacts to lighting
+rather than add surface detail.
+
+This is achieved in two ways:
+
+* Indirect diffuse lighting is made to match global illumination more closely.
+* If specular occlusion is enabled, it is calculated using the bent normals and
+  ambient occlusion instead of just from ambient light.
+  This includes screen-space ambient occlusion (SSAO) and other sources of
+  ambient occlusion.
+
+.. image:: img/spatial_material_bentnormals.webp
+
+Godot only uses the red and green channel of a bent normal map for better
+compression and wider compatibility.
+
+When creating a bent normal map, there are three things required for it to
+work correctly in Godot:
+
+* A **cosine distribution** of rays has to be used when baking.
+* The texture must be created in **tangent space**.
+* The bent normal map needs to use the X+, Y+ and Z+ coordinates, this is
```

Comment:
```suggestion
* The bent normal map needs to use the X+, Y+, and Z+ coordinates, this is
```
Please use [Oxford comma](https://docs.godotengine.org/en/latest/contributing/documentation/docs_writing_guidelines.html#use-the-oxford-comma-to-enumerate-anything).

Reply:
It should be adjusted to follow the style rules, the other case can be fixed separately